qp_meta/sortmerna/sortmerna.py

```diff
@@ -38,10 +38,18 @@
 SORTMERNA_PARAMS = {
     'blast': 'Output blast format',
     'num_alignments': 'Number of alignments',
-    'a': 'Number of threads',
     'm': 'Memory'}
 
 
+# resources per job
```

Comment:
Should these resource requirements be defined outside of the code? I imagine we probably have these defined as globals thoughout all of the plugins.

Reply:
That's correct, they are defined as globals throughout the code for all plugins. Just FYI, this works really well because they can be changed when needed and we don't need to make DB or actual code changes.